composer.json

```diff
@@ -52,6 +52,9 @@
     "wordpress-install-dir": "web/wp"
   },
   "scripts": {
+    "post-root-package-install": [
+            "php -r \"file_exists('.env') || copy('.env.example', '.env');\""
```

Comment:
Updated, please check, feel free to modify in my repo, see [how](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)